README.md

```diff
@@ -18,6 +18,7 @@ The master branch of this project closely follows [GCC development branch][gcc-d
 * [GCC 5.x](https://github.com/D-Programming-GDC/GDC/tree/gdc-5)
 * [GCC 6.x](https://github.com/D-Programming-GDC/GDC/tree/gdc-6)
 * [GCC 7.x](https://github.com/D-Programming-GDC/GDC/tree/gdc-7)
+* [GCC 8.x](https://github.com/D-Programming-GDC/GDC/tree/gdc-8)
```

Comment:
Time to take out 4.9?

Reply:
Looking at the commit history, 4.8 seems to have received the occasional update, so I was under the impression that we'll just drop that.

Any compelling reason to drop 4.9 too?  Such as bug that can't be fixed?

Reply:
1) gdc supports 4 release branches: 5, 6, 7, 8 + master
2) major linux distributions use something between gcc 5 and 7 (even debian stable has gcc 6.3)
3) Maintenance cost for a small team. Merging master into 4.9 often (always) causes nontrivial conflicts. It would be easier to mergr master into other branches more often if the merging isn't difficult.
4) I doubt someone does care about D support in 4.9. I saw questions about gcc7, I'm using gcc5 and gcc4.9 isn't mentioned anymore in the latest issues/PRs.

Reply:
OK then, removed. :-)

Reply:
:+1: